### ISSUE TITLE
Which params in StarDist2D Config are implemented?

### ISSUE THREAD

**Issue by GFleishman** [mpicbg-csbd/stardist#34]
Hi!

Of course - thanks again for the great tools!
I noticed that, of the member variables printed with `vars(config)` only a subset of them have documentation in the doc string for the class. If something is missing from the doc string, does that mean it is not implemented? I'm specifically interested in dropout and batch_norm. I don't see `self.config.unet_dropout` or `self.config.unet_batch_norm` anywhere in model2d.py, or is the config object passed to some keras superclass where dropout and batch norm are considered or something like that?

**Comment by maweigert** [comment]
Hi @GFleishman,

Yes, they will be used - it's a little bit hidden in the source code though: For the 2D model, every config variable that starts with `unet_` will be stripped of that prefix and then passed to `csbdeep.internals.nets.unet_block`. E.g. `config.unet_batch_norm`  will be passed as `batch_norm`, `config.unet_drop` as `dropout` etc.

If you do the following:

```python
from stardist.models import StarDist2D, Config2D

conf = Config2D(unet_batch_norm=True, unet_dropout=.3)

model = StarDist2D(conf, name=None, basedir=None)   

model.keras_model.summary()        
```
you will see that both BatchNorm and DropOut layers will be part of the model.

Hope that helps!

**Comment by GFleishman** [comment]
Yep, completely answers my question, thanks!
